### ISSUE TITLE
`frozenset.copy` shouldn't return a new object.

### ISSUE THREAD

**Issue by DimitrisJim** [RustPython/RustPython#2877]
## Feature

CPython:

```python
>>> f = frozenset(range(1000))
>>> f2 = f.copy()
>>> f is f2
True
```

RustPython:

```python
>>>>> f = frozenset(range(1000))
>>>>> f2 = f.copy()
>>>>> f is f2
False
```

Calling `copy` on a `frozenset` should return itself and not a new object.

**Comment by DimitrisJim** [comment]
@whgusdn321 do you want to handle this?

**Comment by Snowapril** [comment]
@whgusdn321 There is difference in `id` of frozenset copy between rustpython and cpython.
#### rustpython
``` python
>>>>> a = frozenset(range(100))
>>>>> b = a.copy()
>>>>> id(a)
94675967765472
>>>>> id(b)
94675967759248
>>>>> a is b
False
```

#### cpython
```python
>>> a = frozenset(range(100))
>>> b = a.copy()
>>> id(a)
140340461132704
>>> id(b)
140340461132704
>>> a is b
True
```

I think `copy` method work of PyFrozenSet need to be modified.

**Comment by youknowone** [comment]
I think this issue doesn't fit for bare new comer. It requires understanding of `PyRef` which doesn't fit to common sense of rust programmer.

**Comment by Snowapril** [comment]
I fixed this issue in #2883 by fixing `copy` method in `PyFrozenSet`
